Account for %guest cpu time

olivielpeau opened the pull request:
Rebased https://github.com/DataDog/dd-agent/pull/1656 on current master

Thanks to @rhormaza for contributing!

yannmh commented on checks/system/unix.py:
```diff
@@ -587,8 +587,8 @@ def check(self, agentConfig):
+            data = {'cpuUser': us, 'cpuSystem': sy, 'cpuWait': wa, 'cpuIdle': idle, 'cpuStolen': st, 'cpuGuest': guest}
             for key in data.keys():
                 if data[key] is None:
                     del data[key]
```
Unrelated to your changes, and nitpick, but this can be simplified:

```python
data = dict((k,v) for k,v in data.iteritems() if v)
```
(Python 2.6+ compatible)

olivielpeau replied:
We do want to keep `0` values, so I guess it'd rather be:
```python
data = dict((k,v) for k,v in data.iteritems() if v is not None)
```

right ?

Also I personally prefer keeping `del` as it doesn't recreate a new dict but just deletes the entry in the existing dict. But the dict is very small here so I guess it makes no difference.

yannmh commented:
Thanks for the addition @olivielpeau. While you are on it, can you reword the git commit title to match our guidelines please ?

olivielpeau commented:
@yannmh Thanks for the review! I've reworded the commit message and addressed your comment, could you take one last look at my changes please? :smiley:

olivielpeau commented:
CI passed, merging!
